universal-login-contracts/test/utilsTests.ts

```diff
@@ -26,6 +27,13 @@ describe('Tools test', async () => {
     expect(utils.verifyMessage(message, signature)).to.eq(wallet.address);
   });
 
+  it('Should concatenate two bytes arrays', async () => {
```

Comment:
I would add tests (and implementation) for cases when `bytes1` or `bytes2` doesn't start with 0x or is not a hex. Throwing an exception I would consider proper behavior.